### PR TITLE
[REF]8.0 migration views

### PR DESCRIPTION
**Pull request by flotho**

### CODE REVIEW

File: prestashoperpconnect/views/product_view.xml [hunk 1]
```diff
@@ -91,7 +79,7 @@
         <record id="product_connector_presta_tree_view" model="ir.ui.view">
             <field name="model">prestashop.product.template</field>
             <field name="arch" type="xml">
-                <tree string="prestashop product" create="0">
+                <tree string="prestashop product">
```
Comment (pedrobaeza) [inline prestashoperpconnect/views/product_view.xml]: Don't remove the create="0". The goal is to not let create from there any record.

File: prestashoperpconnect/views/product_view.xml [hunk 2]
```diff
@@ -81,8 +71,6 @@
                         <field name="sale_ok"/>
                         <field name="quantity" string="qty"/>
                         <field name="always_available"/>
-                        <field name="available_for_order"/>
```
Comment (pedrobaeza) [inline prestashoperpconnect/views/product_view.xml:84]: Why remove this?

**Comment by flotho** [comment]
@pedrobaeza, should be better now

**Comment by pedrobaeza** [comment]
:+1: For not dizzle you, I'll squash lately your commits in one in my branch. Merging...

**Comment by flotho** [comment]
additionnally, it depends on :
- [ ] https://github.com/OCA/connector/pull/119
  Maybe you could merge this one too

**Pull request merged**